seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/Firefox.scala

```diff
@@ -11,6 +11,12 @@ import scala.collection.JavaConversions._
 import scala.collection.mutable.ArrayBuffer
 
 object Firefox extends SeleniumBrowser {
+
+  private final val popHijackedConsoleScript = {
+    "var callback = arguments[arguments.length - 1];" +
```

Comment:
It is not explicit in the documentation, but the documentation says to use it this way.
https://selenium.googlecode.com/svn/trunk/docs/api/java/org/openqa/selenium/JavascriptExecutor.html